working-together/README.md

```diff
@@ -221,8 +221,16 @@ We've compiled some guidelines from various sources to help people feel included
   - Try to be aware of others' comfort level with eye contact.
   - Err on the side of less eye contact.
 
+## Inclusive language guidelines
 
-## Pronoun guidelines
+- **Acronyms**
+  - The use of acronyms makes it hard for someone who doesn't know their meaning to engage in conversations or have total clarity on some code.
+  - Avoid using acronyms in code, conversations, or documentation as much as possible.
+  - If the use of specific acronyms is inevitable for any reason...
+    - Make sure to document their meanings so that everyone can see them anytime.
+    - When mentioning them in conversations, check if people involved in the conversation know what they mean.
+
```

Comment:
Should this be its own section instead of under the Pronouns guidelines section?